__tests__/app.test.js

```diff
@@ -231,6 +231,45 @@ describe("PATCH /api/articles/:article_id", () => {
   });
 });
 
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204: should delete comment", () => {
+    return request(app).delete("/api/comments/2").expect(204);
+  });
+  test("404: not found", () => {
+    return request(app)
+      .delete("/api/comments/9999")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not Found");
+      });
+  });
+  test("400: invalid id type", () => {
+    return request(app)
+      .delete("/api/comments/banana")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+});
+
+describe("GET /api/users", () => {
+  test("200: an array of objects, each object should have username ,name, avatar_url", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const {users} = body
+        expect(users).toHaveLength(4);
```

Comment:
Great to have the length check - the forEach would automatically pass if the array was empty